jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java

```diff
@@ -241,23 +242,23 @@ private void verifyWidsPngsAndGifs(List<String> wids, List<String> pngs, List<St
             Collection<Asset> storedWids = repository.listAssetsRecursively("/", new FilterByExtension("wid"));
             assertEquals(wids.size(), storedWids.size());
             for (String wid : wids) {
-                repository.assetExists(wid);
+                repository.assetExists(wid.replaceAll("\\s","%20"));
```

Comment:
Spacing again. See also 253 and 261 line. It is not wrong, but from my point of view, it is better to keep the same formatting style.